### ISSUE TITLE
[Feature] Layout breaks when text is enlarged

### ISSUE THREAD

**Issue by hildeor** [AtB-AS/mittatb-app#495]
## Origin
[Usability test with Blindeforbundet, 23.09.2020, finding 14.](https://github.com/AtB-AS/innsikt/blob/master/brukertester/2020-09-23_Blindeforbundet.md#14)

> Layout brekker ved ulike grader av skriftforstørring. (Gjelder egentlig hele appen)

## Motivation
_A short description of what user needs or business goals this feature will solve._

## Hypotheses and assumptions
_A list of hypotheses and assumptions we have made about the user or the proposed solution._

## Proposed solution
Design: define how enlargement of text should be handled in the app (different mode, theming, responsive layout... )

**Comment by hildeor** [comment]
Fixed
